src/index.json

```
@@ -11705,7 +11705,7 @@
                     "summary": "Tools for managing ArcData.",
                     "version": "1.5.5"
                 },
-                "sha256Digest": "04d1e53439504551ab4c34df922a89bed3f0fd05e0f482cadd7998bd51b31e93"
+                "sha256Digest": "e8a1df495eddd4f38d8b61039be81f97545ba26d276357c2f459425b52ca7b4a"
```

@swells could you help check the sha value? Thanks.
```suggestion
                "sha256Digest": "a26158b4c26fc2ae2dd2b36b3908f8fbfc0ce87b8edbbbefe420238c527d3537"
```